### PR TITLE
feat(umaverse): expand recommended synths

### PR DESCRIPTION
**Pull request by Gamaranto**
**Motivation**
This PR expands the recommend list in the "You might also like" section: https://app.clubhouse.io/uma-project/story/1955/update-you-might-be-interested-section-to-always-recommend-another-related-product-if-no-other-related-product-recommend

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [x]  All existing tests pass
- [ ]  Untested

### CODE REVIEW

File: utils/contentful.ts [hunk 1]
```diff
@@ -81,10 +84,28 @@ async function getSynth(address: string): Promise<ContentfulSynth> {
...
+  // Fetch synth in the same category
+  const relatedCmsSynths = await getSynthsByCategory(synth.category);
+  // Get the synths state to see if they're expired or not
+  const allRelatedSynths = (
+    await Promise.all(relatedCmsSynths.map(fetchCompleteSynth))
+  ).filter(errorFilter) as Synth<{ type: ContractType }>[];
+  const relevantRelatedSynths = allRelatedSynths.filter((relatedSynth) => {
+    const isExpired =
+      DateTime.now().toSeconds() > Number(relatedSynth.expirationTimestamp);
+    relatedSynth.address !== synth.address && !isExpired;
```
Comment (daywiss) [inline utils/contentful.ts:96]: there needs to be a return here im pretty sure

File: utils/contentful.ts [hunk 2]
```diff
@@ -81,10 +84,28 @@ async function getSynth(address: string): Promise<ContentfulSynth> {
...
+  });
+
+  if (relevantRelatedSynths.length > 0) {
+    return relevantRelatedSynths;
+  }
+  const nextCategoryIdx =
```
Comment (daywiss) [inline utils/contentful.ts:102]: so is the idea if there are no related synths, rather than showing nothing you show the next category?

Comment (Gamaranto) [reply utils/contentful.ts:102]: yes, exactly!

File: utils/contentful.ts (hunk 1 above)
Comment (Gamaranto) [reply utils/contentful.ts:96]: great catch, thanks!

**Review by daywiss — APPROVED** [review]

**Pull request merged**